backend/btrixcloud/emailsender.py

```diff
@@ -7,6 +7,18 @@
 from email.message import EmailMessage
 
 
+def get_invite_expiration_window():
```

Comment:
Hm, do we actually want to tell users this? Most platforms don't include this info, wonder if there's a security reason for this..

Reply:
Good question, I'm not sure! @SuaYoo - do you have thoughts?

Reply:
Now removed